Makefile

```diff
@@ -137,23 +123,13 @@ test-e2e: KUBECONFIG?=$(HOME)/.kube/config
 test-e2e:
 	go test -v -timeout=20m ./test/e2e/ --kubeconfig $(KUBECONFIG)
 
-.PHONY: test-sec
-test-sec:
-	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
-	gosec -severity medium --confidence medium -quiet ./...
-
-############
-# Binaries #
-############
-
-$(EMBEDMD_BIN):
-	@go install -mod=vendor github.com/campoy/embedmd
-
-$(GOBINDATA_BIN):
-	@go install -mod=vendor github.com/jteeuwen/go-bindata/...
-
-$(JB_BINARY):
-	@go install -mod=vendor github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+$(EMBEDMD_BIN): install-tools
+$(GOBINDATA_BIN): install-tools
+$(JB_BIN): install-tools
+$(GOJSONTOYAML_BIN): install-tools
+$(JSONNET_BIN): install-tools
 
-$(GOJSONTOYAML_BINARY):
-	@go install -mod=vendor github.com/brancz/gojsontoyaml
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from hack/tools.go
+	@cat hack/tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
```

Comment:
(nit) `go list -tags tools -json ./hack/. | jq '.Imports[]'` would also give you the list of packages imported by hack/tools.go.

Reply:
Unfortunately it didn't work :(